src/main.h

```diff
@@ -68,13 +68,13 @@ static const unsigned int MAX_REORG_LENGTH = COINBASE_MATURITY - 1;
 static const bool DEFAULT_WHITELISTRELAY = true;
 /** Default for DEFAULT_WHITELISTFORCERELAY. */
 static const bool DEFAULT_WHITELISTFORCERELAY = true;
-/** Default for -minrelaytxfee, minimum relay fee for transactions in zatoshis/kB */
+/** Default for -minrelaytxfee, minimum relay fee rate for transactions in zatoshis per 1000 bytes. TODO(misnamed, this is a rate) */
 static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = 100;
-//! -maxtxfee default
+/** Default for -maxtxfee in zatoshis. */
 static const CAmount DEFAULT_TRANSACTION_MAXFEE = 0.1 * COIN;
-//! Discourage users to set fees higher than this amount (in satoshis) per kB
+/** Discourage users from setting fee rates higher than this in zatoshis per 1000 bytes. */
```

Comment:
Non-blocking nit: I don't think it was worth doing `s/kB/1000 bytes`; they are the same (though I agree that `kB` can be confused with `kiB`, I don't think it really mattered enough to change).

Reply:
To me "kB" is ambiguous, and I would have assumed it meant 1024 bytes if I hadn't checked.